src/collections/_documentation/performance/index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Performance Monitoring
+sidebar_order: 0
+---
+
+Performance monitoring helps developers measure Apdex, Throughput, and trace slow transactions down to the poor performing API call or DB query. Available for both JavaScript and Python SDKs, performance monitoring helps you both diagnose problems and measure your application's overall health. 
+
+Learn more about performance monitoring in:
+
+- [Homepage]({% link _documentation/performance/performance-homepage.md %}): displays a high-level report of application health, including problematic transactions and both Apdex and Throughput graphs
+- [Distributed Tracing]({% link _documentation/performance/distributed-tracing.md %}): diagnose problems and view interactions across your software systems
+- [Metrics]({% link _documentation/performance/performance-metrics.md %}): provides insight into user experience of your application based on custom thresholds
+- [Discover]({% link _documentation/performance/discover/index.md %}): view data across environments with pre-built queries. Customers subscribed to the Team or Business plan can use Discover to view comprehensive information sent to Sentry.
+
+{% include components/platform_content.html content_dir='getting-started' %}
+
+## Navigating Performance Data
+
+Sentry provides several tools to explore your application's performance data and uncover problems.
+
+The **Performance homepage** is our center for monitoring how your application is doing. Here, you can view a list of all of your endpoints and transactions and view a summary of any transaction in your application. Learn more about the [Homepage]({% link _documentation/performance/performance-homepage.md %}).
```

Comment:
```suggestion
The **Performance homepage** is our center for monitoring how your application is doing. Here, you can view a list of all your endpoints and transactions and view a summary of any transaction in your application. Learn more about the [Homepage]({% link _documentation/performance/performance-homepage.md %}).
```

Reply:
removing "of" might help the sentence flow more smoothly

Reply:
thank you.